src/Interactor/RequestModel.java

```diff
@@ -0,0 +1,12 @@
+package Interactor;
+
+import java.util.*;
+
+/**
+ * 
+ */
+public enum  RequestModel {
```

Comment:
You probably got RequestModel concept wrong )
Request model  is just a class which represents some configuration parameters for analysis.
Analyzer's analyze() method accepts  RequestModel as a parameter and returns a Report.
Object of RequestModel is created by Controller's client